travis/test_flake8

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -v
+# set -v  # verbose
```

Comment:
I decided to comment out instead, to make it easier to re-enable when developing new features.